CHANGELOG.md

```diff
@@ -35,8 +49,10 @@ All notable changes to this project will be documented in this file. This change
 * #168 Bumped to new versions of clojure and tools.cli dependencies
 * #171 Update core.logic to avoid exception from spec
 
-## [0.1.2] / 2015-04-21
+## [0.1.2] - 2015-04-21
 ### Additions
 * Clojurescript/Cljx support (cljc support coming soon). This just works™, kibit will pick up your source paths from your `project.clj`'s `:source-paths`, `[:cljsbuild :builds]`, and `[:cljx :builds]`.
 * Non-zero exit codes. Kibit now exits non-zero when one or more suggestions are made. This is particularly useful for those running checks in a CI environment.
 * You can now run kibit on any Clojure project without a project.clj file. Just call `lein kibit` with any number of files and folders and it will inspect the Clojure files contained within.
+
+[Unreleased]: https://github.com/gorillalabs/kibit/compare/v0.3.0...HEAD
```

Comment:
Can you change gorrillalabs -> jonase here, and in the other places it was used?